packages/fx-core/src/component/resource/botService/messages.ts

```diff
@@ -51,4 +51,19 @@ export class Messages {
   public static readonly CheckOutputLogAndTryToFix = getLocalizedString(
     "plugins.bot.CheckLogAndFix"
   );
+
+  public static readonly NotAllowedToAcquireBotFrameworkToken: [string, string] = [
```

Comment:
Maybe create a function that return this struct will be better.
```
let localString = (s: string) => { return [getDefaultString(s), getLocalizedString(s)]; }
```

Reply:
Good idea